piet-wgsl/examples/winit/Cargo.toml

```diff
@@ -1,5 +1,5 @@
 [package]
-name = "winit"
+name = "winit-demo"
```

Comment:
Ha, I've been meaning to fix this. I presume you ran into the issue where it couldn't work out which `winit` you meant in `cargo run -p winit`?